cmd/updater/update.sh

```diff
@@ -201,14 +201,14 @@ function get_updater_url() {
 
     # the updater will auto-update itself to the latest version, this means that the version of updater that is downloaded
     # can be arbitrary as long as the self-updating functionality is working, hence the hard-coded version
-    UPDATER_FILENAME="install_beta_${OS}-${ARCH}_${MIN_VERSION}.tar.gz"
-    UPDATER_URL="https://algorand-releases.s3.amazonaws.com/channel/beta/${UPDATER_FILENAME}"
+    UPDATER_FILENAME="install_stable_${OS}-${ARCH}_${MIN_VERSION}.tar.gz"
```

Comment:
similar to the `MIN_VERSION` addition, what about something like an `UPDATER_CHANNEL` variable that informs whether to use stable/beta etc, could help make things easier to test?

Reply:
updated